torch/csrc/jit/operator_upgraders/version_map.h

```diff
@@ -0,0 +1,51 @@
+#pragma once
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+namespace torch {
+namespace jit {
+
+struct UpgraderEntry {
+  int version_bump;
+  std::string upgrader_name;
+  std::string old_schema;
+};
+
+static std::unordered_map<std::string, std::vector<UpgraderEntry>> operator_version_map(
```

Comment:
```suggestion
const static std::unordered_map<std::string, std::vector<UpgraderEntry>> kOperatorVersionMap(
```
Consider using const since it won't change. Then naming convention will be `kOperatorVersionMap`.

Reply:
What does k in front of the variable stand for?

Reply:
const

Reply:
i see thx